app/main.py

```diff
@@ -1,3 +1,63 @@
+from __future__ import division
```

Comment:
Please, don't leave unused imports in the code

Reply:
Ooops